### ISSUE TITLE
Make the tests structured

### ISSUE THREAD

**Issue by Glutexo** [RedHatInsights/insights-host-inventory#102]
All tests are currently stuffed in two files: [_test_api.py_](
https://github.com/RedHatInsights/insights-host-inventory/blob/142588853c96554443b5782a4b9db42d90d8f84a/test_api.py) and [_test_unit.py_](
https://github.com/RedHatInsights/insights-host-inventory/blob/142588853c96554443b5782a4b9db42d90d8f84a/test_unit.py). As new tests and cases are added with every pull request, it’d be nice to have them more structured.

I suggest putting them in a _/tests_ folder and then possibly into _api_ and _unit_ subfolders. Then I’d put every test case in a single file, or at least keep only the most closely related together.

Any further improvements like splitting into finer categories or utilizing _Pytest_ etc. can be done subsequently as new issues.

**Comment by dehort** [comment]
I agree.  We should certain start moving the tests into a sub-directory to help organize them better.

**Comment by Glutexo** [comment]
#288 is a step towards this.
